tests/tensor_search/test_validation.py

```diff
@@ -293,12 +293,12 @@ def get_good_index_settings():
                 "model": "hf/all_datasets_v4_MiniLM-L6",
                 "normalize_embeddings": True,
                 "text_preprocessing": {
-                    "split_length": 2,
-                    "split_overlap": 0,
-                    "split_method": "sentence"
+                    "splitLength": 2,
+                    "splitOverlap": 0,
+                    "splitMethod": "sentence"
                 },
                 "image_preprocessing": {
```

Comment:
ditto regarding snake_case in `image_preprocessing, text_preprocessing, normalize_embeddings`

Reply:
Yes, it will be fixed in another PR (created index accept `snake_case`)